packages/contractkit/src/wrappers/ReleaseGold.ts

```diff
@@ -396,6 +398,12 @@ export class ReleaseGoldWrapper extends BaseWrapper<ReleaseGold> {
    */
   setLiquidityProvision = proxySend(this.kit, this.contract.methods.setLiquidityProvision)
 
+  /**
+   * Sets the contract's `canExpire` field to `_canExpire`
+   * @param _canExpire If the contract may be expired after `EXPIRATION_TIME`
```

Comment:
nit: if the contract can expire `EXPIRATION_TIME` after the release schedule finishes.